src/twisted/mail/relaymanager.py

```diff
@@ -607,7 +611,7 @@ def notifyDone(self, relay):
         """
         for message in self.manager.managed.get(relay, ()):
             if self.noisy:
-                log.msg("Setting " + message + " waiting")
+                log.msg(b"Setting " + message + b" waiting")
```

Comment:
I think it would be better to convert `message` to unicode  (e.g. by `repr` ?) than to emit the log text as a bytestring.